common/model/background-texture.js

```diff
@@ -1,5 +1,12 @@
 // @flow
 
+export type PrismicBackgroundTexture = {|
+  image: {
+    url: string
+  },
+  name: string
+|}
```

Comment:
We haven't been doing this for other things, we're just using PrismicFragment - do you think we should be doing this?

Reply:
Ah yeah, this is maybe too paranoid. I'll use PrismicFragment instead.